index.html

```diff
@@ -61,70 +64,121 @@ <h1 class="display-3">Bikes for Refugees</h1>
+						</div>
+						<div class="box">
+							<p>Volunteer with us</p>
+						</div>
+
+					</div>
+					
 					<h2 class="heading-underline">Learn more</h2>
-					<div class="articles">
-						<article class="article">
-							<div class="article-title">
-								Why do refugees need bikes?
+					<div class="article-container">
+						<div class="articles">
+
+							<div class="article-box1">
+								<article class="article">
+									<div class="article-title">
+										Why do refugees need bikes?
+									</div>
+									<p class="article-lead">Many refugees are placed in housing in areas where there are few jobs. Bikes provide low-cost transportation so that they can get to work, manage child care and do their shopping.</p>
+									<p class="article-read-more">
+										<a href="#">
+										<span class="fa fa-caret-right"></span>
+										Learn more
+										</a>
+									</p>
+								</article>
 							</div>
-							<p class="article-lead">Many refugees are placed in housing in areas where there are few jobs. Bikes provide low-cost transportation so that they can get to work, manage child care and do their shopping.</p>
-							<p class="article-read-more">
-								<a href="#">
-									<span class="fa fa-caret-right"></span>
-									Learn more
-								</a>
-							</p>
-						</article>
-						<article class="article">
-							<div class="article-title">
-								How can I help?
+						</div>	
+						<div class="articles">
+							<div class="article-box2">
+								<article class="article">
+									<div class="article-title">
+										How can I help?
+									</div>
+									<p class="article-lead">We need lots of bikes and bike accessories! If you have an old bike you don't use, bring it to one of our dropoff events. Or come attend one of our fundraisers.</p>
+									<p class="article-read-more">
+										<a href="#">
+										<span class="fa fa-caret-right"></span>
+										Learn more
+										</a>
+									</p>
+								</article>
 							</div>
-							<p class="article-lead">We need lots of bikes and bike accessories! If you have an old bike you don't use, bring it to one of our dropoff events. Or come attend one of our fundraisers.</p>
-							<p class="article-read-more">
-								<a href="#">
-									<span class="fa fa-caret-right"></span>
-									Learn more
-								</a>
-							</p>
-						</article>
+						
+						</div>
 					</div>
 				</div>
-				<div class="sidebar-content col-lg">
+				
+				<aside class="sidebar-content col-lg">
 					<h2 class="heading-underline">Upcoming events</h2>
+					
 					<div class="articles row">
+
+						<div class=article-container-1>
+							
+							<article class="article">
+								<div class=articles-row>
+										<a href="#">
+											<img class="img-fluid" src="./images/spring-fundraisers_thumbnail.jpg">
+										</a>
+							  			<div>
+
+											<div class=push>
+										<div class="article-title article-title--sidebar">
```

Comment:
the idea here is not to use custom horizontal margins like you do in your `push` class. 
Try and do everything just using the 3 grid classes `container`, `row`,  and `col-...`
You can move these around, add wrappers, remove them - as you wish, but try not to use any custom spacing and sizing styles (horizontal only, you can add top and bottom spacing, and heights).